crates/sui-bytecode-src-verifier/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "sui-bytecode-src-verifier"
```

Comment:
I would suggest maybe `"sui-source-validation"` as a crate name. The term "verifier" is used too much for language definition level checks (the Move bytecode verifier and the Sui verifier) that we wouldn't want to conflate the two

Reply:
+1 to this!

Reply:
done